just-install.json

```diff
@@ -398,6 +398,14 @@
       },
       "version": "5.0.4"
     },
+    "lockhunter": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe",
+        "x86_64": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe"
```

Comment:
:+1: will change. For situations like this, any value in adding a `all` or `both` arch? I realize those make less sense if desktop Windows gets ported to ARM or something like that.